tests/displayio_shared_bindings.py

```diff
@@ -50,6 +50,13 @@ def __init__(self, width: int, height: int, colors: int) -> None:
         self.height = height
         self.colors = colors
         self.data = {}
+        bits = 1
+        while (colors - 1) >> bits:
+            if bits < 8:
+                bits = bits << 1
+            else:
+                bits += 8
+        self._bits_per_value = bits
```

Comment:
This mimics the logic from blinka displayio here: https://github.com/adafruit/Adafruit_Blinka_Displayio/blob/01b274d3a058b8efb7b3c3638511724cbc290a7d/displayio/_bitmap.py#L64-L68

It may ultimately be work spending a little time to try to update these tests to utilize Blinka Displayio instead of this one-off mocked implementations of core classes anywhere possible.